package/environments/production.js

```
@@ -1,66 +1,60 @@
+"use strict";
```

ESLint

[error] 'use strict' is unnecessary inside of modules.

(strict)